common/bitvec/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "bitvec"
```

Comment:
Do we want to rename this to bitvector or something else to make it easy to distinguish between this and the other popular bitvec crate?

Reply:
@kchalkias: libra-bitvec?